common/src/main/webapp/sass/explore.scss

```diff
@@ -2541,7 +2541,13 @@ a.explore-upgrade-link {
   display: flex;
 
   .p-dropdown {
-    min-width: 20ch;
+    min-width: 5ch;
+    max-width: 12ch;
+
+    @media (min-width: lucumaUICommon.$mobile-responsive-cutoff) {
```

Comment:
I'm not sure it would help here, but CSS now supports container queries.

Reply:
Not for this but I'm considering container queries for the tiles